.github/setup.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+WORKSPACE=$(pwd)
+
+## Common definitions for the bioconda build system. 
+MINICONDA_VER=4.6.14
+BIOCONDA_UTILS_TAG=master
+
+if ! type bioconda-utils 2> /dev/null || [[ $BOOTSTRAP == "true" ]]; then
+    echo "Setting up bioconda-utils..."
+
+    # setup channels
+    conda config --system --add channels defaults
+    conda config --system --add channels bioconda
+    conda config --system --add channels conda-forge
+
+    # install bioconda-utils
+    # fix temporary problem with bioconda-utlis not being compatible with the 
+    # latest version of gitdb
+    additional_packages="gitdb2<3"
+    conda install -y $additional_packages --file https://raw.githubusercontent.com/grst/bioconda-utils/$BIOCONDA_UTILS_TAG/bioconda_utils/bioconda_utils-requirements.txt
+    pip install git+https://github.com/grst/bioconda-utils.git@$BIOCONDA_UTILS_TAG
```

Comment:
Is there a reason that you're using your fork of `bioconda-utils`?

Reply:
Looking at the [diff on your fork](https://github.com/bioconda/bioconda-utils/compare/master...grst:master) it looks like it's to replace the `home` of `'bioconda/bioconda-recipes'` with `"grst/modules"` and also to handle more files than `['meta.yaml', 'build.sh']`.

However, both of these things seem to be handled as function arguments, so I wonder if we can run the vanilla `bioconda-utils` with some combination of flags / config without needing to change the source code?

Reply:
While having a quick look it didn't seem that these params are configurable. So for testing I decided to hardcode it in a fork. 
Eventually, I think the way to go is to make it configurable upstream.